server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/pullrequest/PullRequestLabel.java

```diff
@@ -0,0 +1,15 @@
+package de.tum.in.www1.hephaestus.codereview.pullrequest;
+
+import jakarta.persistence.Embeddable;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Embeddable
```

Comment:
So this means that there is no separate table for it am I right?

Reply:
yes, but we could also create an own table for labels if we think that it is necessary. but then we have to handle the labels more